flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -23,6 +23,7 @@
 import java.time.Duration;
 import java.util.List;
 import java.util.Map;
+import org.apache.flink.annotation.Internal;
```

Comment:
Is it proper to use the flink internal annotation?

Reply:
yes, because Iceberg doesn't have such annotation. I have been using it extensively in the FLIP-27 source, which is approved by Ryan Blue.